Update README.md

Minsoek96 opened the pull request:
[리드미 검토 요청]
1. BestPratice 대안 부분은 최대한 간소화 
2. BestPratice 선정 이유에 대해서 작성
3. 배포주소 추가
4. 기술스택 추가
5. 디렉토리 구조 변경 완료

피드백 부탁드립니다.

Minsoek96 committed:
Update README.md

[리드미 검토 요청]
1. BestPratice 대안 부분은 최대한 간소화 
2. BestPratice 선정 이유에 대해서 작성
3. 배포주소 추가
4. 기술스택 추가
5. 디렉토리 구조 변경 완료

피드백 부탁드립니다.

Minsoek96 committed:
Update README.md

[리드미 검토 요청]

1. BestPratice 대안 부분은 최대한 간소화
2. BestPratice 선정 이유에 대해서 작성
3. 배포주소 추가
4. 기술스택 추가
5. 디렉토리 구조 변경 완료
피드백 부탁드립니다.

jypman commented:
@Minsoek96 @AlgeMoya 
1. 배포링크는 텍스트 링크로 남겨놓는 게 직관적이라고 생각합니다.
배포링크가 안 보이길래 빠트렸나 싶었습니다.
배포링크 아래에 이미지가 있는데 이미지 클릭 시 배포 서비스로 이동하는지 처음에 몰랐습니다.

2. 키보드로 추천 검색 이동 시 어떤 이슈가 발생하고 어떻게 해결했는지에 대한 내용도 어필포인트라고 생각했는데 생략하신 이유가 궁금합니다.
키보드로 추천 검색어 이동 후 엔터 클릭 시 해당 추천 검색어 결과 페이지로 이동한다는 것만 추가해주시면 감사하겠습니다

3. tech stack에 msw가 빠져 있네요

Minsoek96 commented:
@jypman 
1. 공감합니다. 수정하겠습니다.
2. 프로젝트를 진행하면서 이슈가 한두개가 아니라 판단하여 일단은 구조적 통일 위해 뺐습니다. 
죄송합니다 이부분을 어떻게 해결 해야할지 모르겠습니다. 해당 부분만 요청하시면 추가하겠습니다.
4. 추가하겠습니다.

피드백 감사합니다.

jypman commented:
@Minsoek96 
네 그럼 1,3 수정해주시면 될 것 같습니다

AlgeMoya commented:
@Minsoek96 확인했습니다. 저도 처음에 열어봤을 때 배포 링크가 없는 줄 알았어서 당황했습니다. 진영님 1, 3 언급해주신 것 외에 추가적으로 피드백해드릴 사항은 따로 없습니다.

Minsoek96 commented:
@jypman @AlgeMoya 
네 확인했습니다. 
이슈 부분은 죄송합니다. 
이슈에 관련해서나 저희가 소통하면서 해결한 문제들을 다루면 정말 좋을거라 생각했지만
시간상에 제약이 걸릴것같습니다 ㅜㅜ
추후에 리드미를 업데이트 하게되면 참조하겠습니다.

이런 이슈가 있었어요!
검색어 입력 후 방향키로 추천 검색어 이동 시 input value에 검색어의 마지막 글자와 추천 검색어가 포함되는 이슈
이슈는 이렇게 해결했어요!
원인 : 크롬에서 한글로 입력 시 글자가 조합중일 때 키보드 이벤트 발생 시 해당 이벤트 핸들러가 두번 호출된 것
해결 :
event.isComposing을 통해 글자가 조합중인지에 대한 여부 확인 가능
event.isComposing이 true일 경우 조합중이기 때문에 추천 검색어로 이동하는 이벤트 핸들러를 종료
false일 경우 추천 검색어로 이동하는 로직이 포함된 키보드 이벤트 핸들러가 한번만 호출되도록 하여 문제 해결

Minsoek96 committed:
Update README.md

fix/ Readme 
피드백 반영하여 수정하였습니다. 
1. 배포링크는 직관적인게 좋을것같다. ok
2. MSW뱃지 누락 ok
3. 잘못된 관리구조 수정

검토 부탁드립니다.

jypman commented:
@Minsoek96 
고생하셨습니다. 머지해도 된다고 봅니다.

Minsoek96 committed:
fix : 리드미 뱃지 수정

msw

AlgeMoya commented:
@Minsoek96 수고 많으셨습니다! 머지 진행하겠습니다!
